README.md

```diff
@@ -619,6 +620,27 @@ setCustomerHash(
 );
 ```
 
+### getDiskSpaceInfo
+
+Return info about how much free disk space there is in the device
+
+-   Available for app versions 11.10 and higher
+
+```typescript
+getDiskSpaceInfo: () => Promise<{availableBytes: number, totalBytes: number}>;
+```
+
+#### Example
+
+```javascript
+import {getDiskSpaceInfo} from '@tef-novum/webview-bridge';
+
+getDiskSpaceInfo().then(({availableBytes, totalBytes}) => { ... });
+```
+
+-   `availableBytes`: number to see available bytes in the device
+-   `totalBytes`: number to see the total bytes available in the device
```

Comment:
I'll remove the "available" part of this description to make it less confuse with de previous one

Reply:
👍